process/block/alteredaccounts/alteredAccountsProvider.go

```diff
@@ -125,7 +125,7 @@ func (aap *alteredAccountsProvider) processMarkedAccountData(
 	userAccount, ok := account.(state.UserAccountHandler)
 	if !ok {
 		log.Warn("cannot cast AccountHandler to UserAccountHandler", "address", encodedAddress)
```

Comment:
we are recording the error as a WARN in the block processors. Suggestion to remove these WARNs and return a wrapped error. Something like:
```GO
return fmt.Errorf("%w while casting to UserAccountsHandler for address %s", errCannotCastToUserAccountHandler, encodedAddress)
```
Valid for this line, L119, L140, L163

Reply:
done